src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -22,14 +27,26 @@
 @RequestMapping("/late-filing-penalty/company/{companyNumber}/penalty/{penaltyId}/confirmation")
 public class ConfirmationController extends BaseController {
 
-    private static final String PPS_CONFIRMATION_PAGE = "pps/confirmationPage";
+    private static final String CONFIRMATION_PAGE = "pps/confirmationPage";
 
     private static final String PAYMENT_STATE = "payment_state";
 
+    private static final String COMPANY_NAME_ATTR = "companyName";
```

Comment:
These `ATTR` variables can be made package-private visibility then they can be imported as static in your ` src/test/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationControllerTest.java` rather than copying them there too.  Otherwise, I don't see much need for moving them into these constant variables as they are only used once in this class.

Reply:
Changed from private to public in order to let the unit test to use the attribute

Reply:
Can you just remove the `public` modifier here?

Reply:
Updated to remove the public modifier